src/sources/community.js

```diff
@@ -123,6 +124,23 @@ class CommunitySource extends Source {
       avatar: `https://github.com/${this.owner}.png?size=200`
     };
   }
+
+  get authzPolicy() {
+    return [
+      {tag: authz.tags.Visibility.Public, role: "*", allow: [authz.actions.Read]},
+    ];
+  }
+  get authzTags() {
+    return new Set([
+      authz.tags.Type.Source,
+      authz.tags.Visibility.Public,
+    ]);
+  }
```

Comment:
Yep. For the authz properties, I opted not to use inheritance by default so that it was crystal clear when looking at each method in isolation exactly what it'd return, without tracing up the parent classes. This also seemed "safer" for sensitive methods, since changes we make to parent classes can make unexpected action-at-a-distance changes to subclasses. In my eye, the tradeoff is different for non-sensitive functionality, which is why the source classes make use of inheritance elsewhere.

Reply:
Ah, okay. I was initially confused that `PrivateGroupSource` didn't have `authzTags` and then I realized it was just inheriting it from the parent class.

Reply:
Yeah, that's an inconsistency I hadn't noticed earlier.